packages/integration-tests/src/e2e-flow/setup_package_manager.ts

```diff
@@ -0,0 +1,120 @@
+import { execa } from 'execa';
+import * as fsp from 'fs/promises';
+import { existsSync } from 'fs';
+import * as path from 'path';
+import * as os from 'os';
+
+type PackageManager = 'npm' | 'yarn-classic' | 'yarn-modern' | 'pnpm';
+type PackageManagerExecutable = 'npx' | 'yarn' | 'pnpm';
+
+const initializeNpm = async () => {
+  {
+    const { stdout } = await execa('npm', ['config', 'get', 'cache']);
+    const npxCacheLocation = path.join(stdout.toString().trim(), '_npx');
+    if (existsSync(npxCacheLocation)) {
+      await fsp.rm(npxCacheLocation, { recursive: true });
+    }
+  }
+};
+
+const initializePnpm = async (packageManager: string) => {
```

Comment:
seems like the function name is saying that packageManager is always going to be 'pnpm' here. Can we remove the function arg?

Reply:
Good catch. Done 17f406b70c69579ad2aa87c69f57018c8c2dd2f9